Sources/AsyncHTTPClient/ConnectionPool/HTTP2/HTTP2ClientRequestHandler.swift

```diff
@@ -193,11 +193,15 @@ final class HTTP2ClientRequestHandler: ChannelDuplexHandler {
         case .forwardResponseBodyParts(let parts):
             self.request!.receiveResponseBodyParts(parts)
 
-        case .failRequest(let error, let finalAction):
+        case .failRequest(let error, _):
```

Comment:
Do we still use `finalAction` elsewhere or can we remove it from the enum?

Reply:
We use it in the HTTP1ConnectionStateMachine.

Reply:
This is the reason for the comment starting in line 200.